WorkloadManagementSystem/Service/WMSAdministratorHandler.py

```diff
@@ -161,9 +174,11 @@ def export_getSiteMaskSummary(self):
 
     return S_OK(siteDict)
 
-##############################################################################
-  types_getCurrentPilotCounters = [ DictType ]
-  def export_getCurrentPilotCounters( self, attrDict={}):
+  def export_getCurrentPilotCounters( cls, attrDict={} ):
+  ##############################################################################
+  types_getCurrentPilotCounters = [ dict ]
+  @classmethod
+  def export_getCurrentPilotCounters( cls, attrDict={} ):
```

Comment:
Something wrong here (line 177 has to be deleted)

Reply:
Yes, I was not feeling well yesterday and I can see it the code - I managed to create conflicts when solving the conflicts ;)